### PR TITLE
IDR omero and proxy configuration and updates (IDR-0.3.0)

### PR DESCRIPTION
**Pull request by manics**
This is most of the setup and configuration for the omero and proxy instances.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1090](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1090/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1090/consoleText) for more details.
Possible conflicts:
  - PR #181 aleksandra-tarkowska 'upgrade mapr'
    - ansible/idr-playbooks/idr-omero.yml
  - PR #188 manics 'IDR hostgroup naming (IDR-0.3.0)'
    - ansible/idr-playbooks/idr-omero.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1091](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1091/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1091/consoleText) for more details.
Possible conflicts:
  - PR #181 aleksandra-tarkowska 'upgrade mapr'
    - ansible/idr-playbooks/idr-omero.yml
  - PR #188 manics 'IDR hostgroup naming (IDR-0.3.0)'
    - ansible/idr-playbooks/idr-omero.yml

File: ansible/idr-playbooks/group_vars/omero-hosts.yml [hunk 1]
```diff
@@ -3,13 +3,14 @@ omero_dbname: idr
...
 omero_datadir: /data/OMERO
 
-# TODO: change this to a lower system uid
-omero_system_uid: "{{ idr_secret_omero_system_uid | default(546) }}"
+omero_system_uid: 546
 
 omero_web_runtime_redis: True
 # This refers to a custom IDR release URL, not an official OMERO version
-omero_release: "0.0.7-rc1"
+omero_release: "0.0.7-rc2"
```
Comment (joshmoore) [inline ansible/idr-playbooks/group_vars/omero-hosts.yml]: omero_release is being bumped elsewhere?

File: ansible/idr-playbooks/files/IDR-OMERO-52-omero.j2 [hunk 2]
```diff
@@ -10,7 +10,7 @@ config set omero.pixeldata.fail_if_missing true
 # web
 config set omero.web.application_server "wsgi-tcp"
 config set omero.web.application_server.max_requests 300
-config set omero.web.wsgi_timeout 30
+#config set omero.web.wsgi_timeout 30
```
Comment (joshmoore) [inline ansible/idr-playbooks/files/IDR-OMERO-52-omero.j2]: Can this be dropped? cc: @aleksandra-tarkowska

File: ansible/idr-playbooks/group_vars/omero-hosts.yml [hunk 3]
```diff
@@ -20,5 +21,58 @@ omero_web_public_password: "{{ idr_secret_omero_web_public_password | default('p
...
 
-omero_logmonitor_slack_token: "{{ idr_secret_omero_logmonitor_slack_token | default(None) }}"
-omero_logmonitor_slack_channel: "#logs"
+
+######################################################################
+# mapr
+
+omero_web_app_packages:
+#  - "omero-mapr==0.1.1"
+  - "git+git://github.com/aleksandra-tarkowska/omero-mapr.git@0.1.6#egg=omero_mapr"
```
Comment (joshmoore) [inline ansible/idr-playbooks/group_vars/omero-hosts.yml]: mapr is being bumped elsewhere?

File: ansible/idr-playbooks/idr-local-files.yml [hunk 4]
```diff
@@ -4,28 +4,25 @@
...
     become: yes
-    file:
-      path: /srv/www/html
-#      recurse: yes
-      serole: _default
-      setype: _default
-      seuser: _default
-      state: directory
+    copy:
+      dest: "/home/omero/OMERO.server/lib/python/omeroweb/webgateway/static/webgateway/img/logo2.png"
```
Comment (joshmoore) [inline ansible/idr-playbooks/idr-local-files.yml]: Something here isn't yet correct, but I assume we handle in a follow up.

File: ansible/idr-playbooks/group_vars/omero-hosts.yml (hunk 1 above)
Comment (manics) [inline ansible/idr-playbooks/group_vars/omero-hosts.yml]: It'll be in a later commit/PR https://github.com/manics/infrastructure/commit/4668d16b3158fcc80fde9209fb9729689b1b18f8

File: ansible/idr-playbooks/group_vars/omero-hosts.yml (hunk 3 above)
Comment (manics) [inline ansible/idr-playbooks/group_vars/omero-hosts.yml]: It'll be in a later commit/PR https://github.com/manics/infrastructure/commit/4668d16b3158fcc80fde9209fb9729689b1b18f8
https://github.com/manics/infrastructure/commit/4c11c5cf6f37e40fa44ba452c637f15428b6e064

**Comment by joshmoore** [comment]
Ok. Merging and looking forward to follow-ups.

**Pull request merged**